src/components/ExportAccountDialog.js

```diff
@@ -13,6 +13,11 @@ import { useWallet } from '../utils/wallet';
 export default function ExportAccountDialog({ open, onClose }) {
   const wallet = useWallet();
   const [isHidden, setIsHidden] = useState(true);
+  const [isArrayFormat, setArrayFormat] = useState(false);
+
+  const keyOutput = isArrayFormat
+    ? `[${[].slice.call(wallet.provider.account.secretKey)}]`
```

Comment:
Syntactically `Array.from(wallet.provider.account.secretKey)` is a lot more clear here IMO. Another alternative would be `[...wallet.provider.account.secretKey]`.

`[].slice.call()` is the pre-es6 way to do this and is confusing.

Reply:
Thanks, I made the change for this.